media-rs-services/src/main/java/nl/vpro/rs/media/AuthorityRestService.java

```diff
@@ -0,0 +1,48 @@
+package nl.vpro.rs.media;
+
+import javax.ws.rs.*;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+
+import org.jboss.resteasy.annotations.providers.multipart.XopWithMultipartRelated;
+
+import nl.vpro.domain.media.update.MediaUpdate;
+import nl.vpro.domain.media.update.UpdateSupplier;
+import nl.vpro.jmx.Description;
+
+import static nl.vpro.rs.media.MediaBackendRestService.ERRORS;
+import static nl.vpro.rs.media.MediaBackendRestService.ID;
+
+/**
+ * @author Michiel Meeuwissen
+ * @since 7.7
+ */
+@Path("/authority")
+@Consumes({MediaType.APPLICATION_XML, MediaType.APPLICATION_JSON})
+@Description("Services on https://api.poms.omroep.nl/authority")
```

Comment:
Wil je de base-URL ook in deze description hebben? Die kan natuurlijk per omgeving verschillen

Reply:
Het is in feite onzin, die annotatie is voor jmx. Ik verplaatst het dnaar de javadoc.